imagehash/__init__.py

```diff
@@ -116,10 +116,10 @@ def average_hash(image, hash_size=8):
 def phash(image, hash_size=32):
```

Comment:
Sure thing.

You know, I'm fluent if rusty in C++, but honestly I found their code a little hard to read.  It might be interesting to try porting some of their other algorithms, but I think it would be a fair bit of work to do it.